Packs/SekoiaXDR/Integrations/SekoiaXDR/SekoiaXDR.py

```diff
@@ -629,92 +754,147 @@ def get_remote_data_command(
         )
-        entries = [
-            {
-                "Type": EntryType.NOTE,
-                "Contents": {"dbotIncidentReopen": True},
-                "ContentsFormat": EntryFormat.JSON,
-            }
-        ]
 
-    demisto.debug("#### Leaving MIRRORING IN - get_remote_data_command ####")
+        # Close the XSOAR incident using mirroring
+        if (
+            (close_incident)
+            and (alert_status in ["Closed", "Rejected"])
+            and (investigation["status"] != 1)
+        ):
+            demisto.debug(
+                f"Alert {alert_short_id} with status {alert_status} was closed or rejected in Sekoia, closing incident {incident_id} in XSOAR"  # noqa: E501
+            )
+            entries = [
+                {
+                    "Type": EntryType.NOTE,
+                    "Contents": {
+                        "dbotIncidentClose": True,
+                        "closeReason": f"{alert_status} - Mirror",
+                        "closeNotes": close_note,
+                    },
+                    "ContentsFormat": EntryFormat.JSON,
+                }
+            ]
+
+        # Reopen the XSOAR incident using mirroring
+        if (
+            (reopen_incident)
+            and (alert_status not in ["Closed", "Rejected"])
+            and (investigation["status"] == 1)
+        ):
+            demisto.debug(
+                f"Alert {alert_short_id} with status {alert_status} was reopened in Sekoia, "
+                f"reopening incident {incident_id} in XSOAR"
+            )
+            entries = [
+                {
+                    "Type": EntryType.NOTE,
+                    "Contents": {"dbotIncidentReopen": True},
+                    "ContentsFormat": EntryFormat.JSON,
+                }
+            ]
+
+        if not alert.get("job_uuid"):
+            # If the job uuid is not present, we will return the alert and the entries
+            demisto.debug("#### Leaving MIRRORING IN - get_remote_data_command ####")
+            return GetRemoteDataResponse(mirrored_object=alert, entries=entries)
+        else:
+            # If the job uuid is present, we will add the alert to the context
+            context_cache = get_integration_context() or {}
+            mirroring_cache = context_cache.get("mirroring_cache")
+            alert_dict = {"alert": alert, "entries": entries}
+            if not mirroring_cache:
+                cached_alerts = []
+                cached_alerts.append(alert_dict)
+                context_cache["mirroring_cache"] = cached_alerts
+                demisto.debug(
+                    f"mirroring cache in the first condition : {context_cache}"
+                )
+                set_integration_context(context_cache)
+            else:
+                new_context = mirroring_cache.append(alert_dict)
```

Comment:
Note that `new_context` will always be equal to `None`, because append does not return the new list, but only updates the `mirroring_cache`.